CHANGELOG.md

```diff
@@ -0,0 +1 @@
+- Enable single builds for 2nd gen functions. (#6275)
```

Comment:
This feature is a big deal! I'd hype this up a bit more and add some context around potential perfomance improvements - ie:
```suggestion
- Improved performance  and reliability when deploying multiple 2nd gen functions. (#6275)
```